dipy/core/sphere.py

```diff
@@ -498,6 +500,155 @@ def disperse_charges(hemi, iters, const=.2):
     return HemiSphere(xyz=charges), potential
 
 
+def _equality_constraints(vects, *args):
+    """Spherical equality constraint. Returns 0 if vects lies on the unit
+    sphere.
+
+    Parameters
+    ----------
+    vects : array-like shape (N * 3)
+        Points on the sphere.
+    Returns
+    -------
+    array-like (N,)
+        Difference between squared vector norms and 1.
+
+    """
+
+    N = vects.shape[0] / 3
```

Comment:
This may cause issues with Python 3, you may replace the division operator with // (here and in the remaining of the code).

Reply:
Nice catch. Thanks. Fixed in f208032.